test/fixtures/r4/Luna60_McCullough561_6662f0ca-b617-4e02-8f55-7275e9f49aa0.json

```diff
@@ -1687,6 +1687,9 @@
       "resource": {
         "resourceType": "Condition",
         "id": "9934bc4f-58af-4ecf-bb70-b7cc31987fc5",
+        "meta": {
+          "profile": ["http://hl7.org/fhir/us/core/StructureDefinition/us-core-condition-encounter-diagnosis"]
+        },
```

Comment:
I kind of feel like a jerk saying this, since the test will pass either way, but... technically [US Core Condition Encounter Diagnosis](http://hl7.org/fhir/us/core/STU5/StructureDefinition-us-core-condition-encounter-diagnosis.html) requires that the `category` include `encounter-diagnosis`.  So can you add the category so it is a valid instance of the US Core profile it claims?
```
"category" : [
  {
    "coding" : [
      {
        "system" : "http://terminology.hl7.org/CodeSystem/condition-category",
        "code" : "encounter-diagnosis",
        "display" : "Encounter Diagnosis"
      }
    ]
  }
],
```

Reply:
hahaha no I appreciate this. And now I know what you're thinking, yes I did change the code used in the DSTU2 fixture when I added the argonaut profile to make sure it was actually in the valueset heheheh